gsw/geostrophy.py

```diff
@@ -105,6 +106,47 @@ def geo_strf_dyn_height(SA, CT, p, p_ref=0, axis=0, max_dp=1.0,
     return dh
 
 
+@match_args_return
+def geo_strf_steric_height(SA, CT, p, p_ref=0, axis=0, max_dp=1.0, interp_method="pchip"):
+    """
+    Steric height anomaly as a function of pressure.
+
+    Parameters
+    ----------
+    SA : array-like
+        Absolute Salinity, g/kg
+    CT : array-like
+        Conservative Temperature (ITS-90), degrees C
+    p : array-like
+        Sea pressure (absolute pressure minus 10.1325 dbar), dbar
+    p_ref : float or array-like, optional
+        Reference pressure, dbar
+    axis : int, optional, default is 0
+        The index of the pressure dimension in SA and CT.
+    max_dp : float
+        If any pressure interval in the input p exceeds max_dp, the dynamic
+        height will be calculated after interpolating to a grid with this
+        spacing.
+    interp_method : string {'mrst', 'pchip', 'linear'}
+        Interpolation algorithm.
+
+    Returns
+    -------
+    steric_height : array
+        This is the integral of specific volume anomaly with respect
+        to pressure, from each pressure in p to the specified
+        reference pressure divided by the constant value of the gravitational 
```

Comment:
Good condensation. Slight tweaks:
"pressure divided by the constant value..." -> "pressure, divided by the global mean surface value..."
and append "(see page 46 of Griffies, 2004)" to the end of the sentence, or maybe the end of the docstring.  If numpydoc has a 
"References" section, add the Matlab doc ref for Griffies to it; otherwise, stuff into the parenthetical addition above, or tack it on the end of the docstring without its own section.